dll/win32/kernel32/client/sysinfo.c

```diff
@@ -380,13 +380,27 @@ GetNumaProcessorNode(IN UCHAR Processor,
     return TRUE;
 }
 
-/*
- * @implemented
- */
+/**
+ *
+ * @name GetNumaAvailableMemoryNode
+ * @implemented NT 5.2
+ *
+ *     Retrieves the amount of memory available in the specified node.
+ *
+ * @param Node
+ *        The number of the node.
+ *
+ * @param AvailableBytes
+ *        The amount of available memory for the node, in bytes.
+ *
+ * @return TRUE or FALSE on error.You can get extended information using the GetLastError function.
+ *
+ **/
 BOOL
 WINAPI
-GetNumaAvailableMemoryNode(IN UCHAR Node,
-                           OUT PULONGLONG AvailableBytes)
+GetNumaAvailableMemoryNode(
+    IN UCHAR Node,
+    OUT PULONGLONG AvailableBytes)
```

Comment:
Why this change?

Reply:
As ThFabba says, keeping a PR in scope will make it easier to review, and reduces the likelyhood of unnecessary discussion which can prolong an issue.  Also, I'm not sure where, but I remember reading a recommendation that if you make style only changes to a file, they should be in a separate commit from functional changes.